lua/nvim-tree/api.lua

```diff
@@ -175,12 +175,12 @@ end
 
 ---@param mode string
 ---@return fun(node: table)
-local function open_or_expand_or_dir_up(mode)
+local function open_or_expand_or_dir_up(mode, toggle_group)
   return function(node)
     if node.name == ".." then
       actions.root.change_dir.fn ".."
     elseif node.nodes then
-      lib.expand_or_collapse(node)
+      lib.expand_or_collapse(node, toggle_group)
     else
       edit(mode, node)
```

Comment:
The other usages of this are for "open or edit" functionality.

This one is a toggle; falling back to edit is not intuitive. Let's do nothing on files please.